sql/src/main/java/io/crate/operation/aggregation/impl/TDigestState.java

```diff
@@ -84,11 +83,8 @@ public static TDigestState read(StreamInput in) throws IOException {
         return state;
     }
 
-    @Override
-    public void add(TDigest other) {
+    public void add(TDigestState other) {
```

Comment:
it is different methods signatures, you cannot override 
 public void add(TDigest other) 
with 
 public void add(TDigestState other)